### PR TITLE
Adapted to new operator structure (as far as possible).

### PR DESCRIPTION
**Pull request by maltesander**
## Description

Updated to operator-rs 0.7.0.
Removed packaging and operator crate.
Updated docs.
Fixed port on 3030 (no CLI anymore).

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by razvan** [comment]
I think the operator should create a service for http and just use standard ports.

File: examples/service/regorule-service.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: regorule-service
+spec:
+  type: ClusterIP
```
[the remark this answers is not in the document]
Comment (razvan) [reply examples/service/regorule-service.yaml:6]: Decided to leave it as is and manually configure the OPA operator for now.

**Review by razvan — APPROVED** [review]
works.

**Comment by maltesander** [comment]
bors merge

**Pull request merged**